run-orachk.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+orachk_dir=$(dirname $0)
+orachk_env_file=${orachk_dir}/orachk.sh
+
+[[ -f $orachk_env_file ]] || {
+    echo
+    echo $orachk_env_file not found
+    echo
+    exit 1
+}
+
+. $orachk_env_file
+
+# use of undeclared variables is fatal
+set -u
+
+[[ -x $RAT_CRS_HOME ]] || {
```

Comment:
nit:  $RAT variables might be confused with real application testing

Reply:
Yes, that is true, and I also find it annoying.
These are however Oracle's variables for ORAchk and EXAchk.
Example
https://docs.oracle.com/cd/E68491_01/OEXUG/troubleshooting-orachk-and-exachk.htm#OEXUG-GUID-C73011FD-6AD7-4540-B4B7-8E42C57FA604

Reply:
Ahh got it.  Technical debt, I guess.